CHANGELOG.md

```diff
@@ -1,4 +1,109 @@
 # Changelog
+* Friday 20th May 2022
+v0.16.2
+
+## Bug fixing
+* We noticed in testing some additional latency caused by our feature flag implementation - so we refactored this to make it much faster. This was mostly visible in our custom voice implementations. 
+
+### Enhancements
+* We added the ability to share audio with other users. It's a magic link feature. 
+For example
+```shell
+curl -X POST \
+  'https://v1.api.audio/mastering' \
+  --header 'x-api-key: API-KEY' \
+  --header 'Content-Type: application/json' \
+  --data-raw '{
+    "scriptId": "xx",
+    "share": true
+}'
+```
+This returns a url - if you goto 
+ 
+ https://console-api.audio/share?id=your_id
+ 
+ You'll get a page that you can share with your organisation and via Whatsapp and other social 
+ networks. 
+
+* We've updated our [voice library page](https://library.api.audio/voices) to have faster loading and we've added some other improvements in the GET request for voices so you can have a better voice discovery experience. Most of this is backend and performance enhancements. 
+* Enhanced query parameters for voice discoverability
+
+`GET https://v1.api.audio/voice?tags=upbeat,storytelling`
+
+Will voices that have both of these tags. We also introduced pagination as well. 
+
+**Chunking**
+
+Use parameter `limit` to set the amount of returned voices. Use `offset` to iterate through the results.
+
+E.g.
+
+`GET https://v1.api.audio/voice?limit=10&offset=10`
+
+`GET https://v1.api.audio/voice?limit=10&offset=20`
+
+`GET https://v1.api.audio/voice?limit=10&offset=30`
+
+
+**Query body in POST /voice**
```

Comment:
FYI This is not available in the SDK yet, only through HTTP request.

Reply:
That's ok